server/config/pipeline.js

```diff
@@ -73,7 +72,21 @@ function setUpPipelines(config, args, pathCascade) {
                     },
                     queue: true
                 });
-            } else {
+            } 
+            else if(key == "barcode_strand_match"){
+                checkPipeline(config, key, pipeline);
+                if (pipeline.ignore) return;
+                parseAnnotationRequires(pipeline, config, pathCascade, args) 
+                pipeline.configOptions = mergeAdditionalAnnotationOptions(pipeline.configOptions, config, args);
+                
+                // set up the runner
+                pipelineRunners[key] = new PipelineRunner({
+                    config: pipeline,
+                    onSuccess: () => {global.NOTIFY_CLIENT_DATA_UPDATED();},
+                    queue: true
+                });
+            }
+            else {
```

Comment:
keep style of old code. Originally it was `} else {`. Same for the else if above

Reply:
okay.